src/client/pages/PlaceList/PlaceList.jsx

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import axios from 'axios';
+import PlaceCard from '../../components/PlaceCard/PlaceCard'
+import DummyPlaces from '../../components/PlaceCard/DummyData'
+import PlaceListTopBar from '../../components/AppBarTop/PlaceListTopBar';
+
+const DummyPlace = {
+  place_id: 1,
+  name: "Ya Kun Kaya Toast",
+  address: "1 Fusionopolis Way, #B1-15, S'138632",
+  image_link: null,
+  voteCount: {
+    LIKE: 5,
+    DISLIKE: 0
+  }
+};
+
+class PlaceList extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+        places: [],
+        isDoneFetching: false,
+        isLoading: true
+    };
+  }
+
+  routeChange(pathname, placeId) {
+    this.props.history.push({
+      pathname,
+      state: {
+        placeId
+      }
+    });
+  }
+
+  /*
+  componentDidMount() {
+    let instance = this;
```

Comment:
The requests need to be implemented right?

Reply:
Eventually, but for now I'm just pulling from demo data embedded in the files

Reply:
This is the list of places they left and right swiped on. This has to come from the database right? Or do we have another way to source it?